app/trans/http/handler.go

```diff
@@ -0,0 +1,31 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/go-devs-ua/octagon/app/ent"
+)
+
+type User = ent.User
+
+// ApiHandler is User HTTP handler
+type ApiHandler struct {
```

Comment:
We can't name it as "ApiHandler".
First of all, we have tot follow naming rules - "API" or "api" but not "Api".

Apart of it, it's obviously, that's it's API, cos we are already inside the transport/http pkg.
Maybe, it would be better to call it as "UserHandler" for now.

Reply:
I agree with you. Will fix it